.github/workflows/linux-ci.yml

```diff
@@ -13,14 +13,33 @@ on:
 jobs:
   build-test:
     runs-on: ubuntu-latest
+    env:
+      DOTNET_MULTILEVEL_LOOKUP: 1
 
     strategy:
       matrix:
-        version: [netcoreapp3.1,net6.0]
+        version: [netcoreapp3.1,net6.0,net7.0]
 
     steps:
     - uses: actions/checkout@v3
 
+    - uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: '3.1.x'
+
+    - uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: '5.0.x'
```

Comment:
Do you need to support 5.x? It is going to be not supported next month `May 2022`.

Reply:
We do not - we will be removing support for that (In Separate PR). These changes are done due a known issue with GH actions